src/__mocks__/models/vaults.ts

```diff
@@ -0,0 +1,24 @@
+import BigNumber from 'bignumber.js';
+
+import { Vault } from 'types';
+
+export const vaults: Vault[] = [
```

Comment:
I would recommend keeping a mock of raw api responses and then modifying those here

Reply:
It would be a nightmare to do. Just check the `useGetVaults` hook: we get the data from multiple smart contracts and format it accordingly.

Reply:
I don't think it is difficult to do and I think it is a nice organized setup for mocks that gives us the data we need for testing.